src/gateway/get-started/key-authentication.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Key Authentication
+content-type: tutorial
+book: get-started
+chapter: 5
+---
+
+Authentication is the process of verifying that a requestor has permissions to access a resource. 
+As it's name implies, API Gateway authentication, authenticates the flow of data to and from your upstream services. 
+
+{{site.base_gateway}} has a library of plugins that support 
+the most widely used [methods of API gateway authentication](/hub/#authentication). 
+
+Common authentication methods include:
+* Key Authentication
+* Basic Authentication
+* OAuth 2.0 Authentication
+* LDAP Authentication Advanced
+* OpenID Connect
+
+## Authentication benefits
+
+With {{site.base_gateway}} controlling authentication, requests won't reach upstream services unless the client has successfully
+authenticated. This means upstream services will be processing pre-authorized requests, freeing them from the 
+cost of authentication, which is a savings in compute time *and* development effort.
+
+{{site.base_gateway}} has visibility into all authentication attempts, which provides the ability to build 
+monitoring and alerting capabilities supporting service availability and compliance. 
+
+For more information, see [What is API Gateway Authentication?](https://konghq.com/learning-center/api-gateway/api-gateway-authentication).
+
+## Enable authentication
+
+The following tutorial walks through enabling of the [Key Authentication plugin](/hub/kong-inc/key-auth/) across
+various aspects in {{site.base_gateway}}.
+
+API key authentication is a simple and popular method for enforcing API authentication. In key authentication, 
+{{site.base_gateway}} is used to generate and associate an API key to a [consumer](/gateway/latest/admin-api/#consumer-object). 
+That key is the authentication secret presented by the client when making subsequent requests. {{site.base_gateway}} approves or 
+denies requests based on the validity of the presented key. This process can be applied globally or to individual 
+[services](/gateway/latest/admin-api/#service-object) and [routes](/gateway/latest/admin-api/#route-object).
```

Comment:
For "consumers" (above), "services", and "routes", should those link to the key concepts pages instead of the admin API?

Reply:
Sounds good, would you like to suggest the change? Or I can find the right links

Reply:
@cloudjumpercat updated the links in latest